bokehjs/src/lib/models/callbacks/open_url.ts

```diff
@@ -25,13 +26,17 @@ export class OpenURL extends Callback {
 
     this.define<OpenURL.Props>({
       url: [ p.String, 'http://' ],
+      same_tab: [ p.Boolean, false ],
     })
   }
 
   execute(_cb_obj: unknown, {source}: {source: ColumnarDataSource}): void {
     const open_url = (i: number) => {
       const url = replace_placeholders(this.url, source, i)
-      window.open(url)
+      if (this.same_tab)
+        window.open(url)
+      else
+        window.location.href = url
```

Comment:
I fixed the merge conflict, however, I think this logic should be reversed.

Reply:
Ooups, fixed now.